### ISSUE TITLE
Problem (no crash) with the mod[Tree Harvester]

### ISSUE THREAD

**Issue by juanretamales** [Serilum/.issue-tracker#306]
this is for the normal use.

1- When i chopping a tree of base 2x2 how to dark oak, the mod not replace sapplings (or any sapling of this tree)
 in random ocation this replaccing work but i dont know when work or not work....

2- the axe (i tested with stone axe) not broke, por example, i can use infinite this axe with 1 of duration.

this is all for now

**Comment by afzalarsalan** [comment]
I can confirm both of these issues, the sapling replace doesn't seem to be 100% even when I can see sapling dropped on the floor and the durability break seems to be pretty random once the axe gets to an extremely low threshold.

**Comment by Zygus42** [comment]
I chopped a tons of trees with an unenchanted golden axe with 0 durability.  It lost durability correctly until it got to 0, but then I was able to keep using it.

And yes - the replacement of saplings seem very haphazard - seems to work mostly ok with birch trees (except for the 1st one when you have no saplings in inventory), but doesn't seem to work at all with Biome O Plenty Redwoods of any size.

treeharvester_1.16.4-2.0
